npy_array/array.BUILD

```diff
@@ -0,0 +1,11 @@
+cc_library(
```

Comment:
Can we place these <dep>.BUILD files in an appropriate folder such as /deps?

Reply:
Done.